en/README.md

```diff
@@ -0,0 +1,1138 @@
+|:-----------------|:--|:-:|:----------:|:---|
+|hostId  |String255|M|-|Host Id of monitered by monitoring server.|
+|hostName|String255|M|-|Host name of monitored by monitoring server.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "lastInfo": "201504091052",
+    "updateType": "UPDATED",
+    "hosts": [
+      {
+        "hostName": "exampleHostName1",
+        "hostId": "1"
+      }
+    ]
+  },
+  "method": "putHosts",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### putHostGroups(method)
+
+Send all host groups information to Hatohol server to use "ALL" option when complate conect to Hatohol server and changed hosts information that saved in internal.
+
+ In case of using "UPDATE" option, send diffarence to use lastInfo that got using [getLastInfo](#user-content-getlastinfomethod) or saved in internal to Hatohol server.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostGroups  |object array|M|-|This is object array that stores host groups information. Look at the following table for details.|
+|updateType|string    |M|-|Select sending option that match the situation from [[List](#user-content-updatetype)].|
+|lastInfo    |String32767 |O|-|Insert to here host group information of last. This information is result of [getLastInfo](#user-content-getlastinfomethod).|
+
+***hostGroups object***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|groupId  |String255|M|-|Id of host group.|
+|groupName|String255|M|-|Host grou name of compliant the above group Id.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "lastInfo": "201504091049",
+    "updateType": "ALL",
+    "hostGroups": [
+      {
+        "groupName": "Group2",
+        "groupId": "1"
+      }
+    ]
+  },
+  "method": "putHostGroups",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### putHostGroupMembership(method)
+
+Send all host group membership information to Hatohol server to use "ALL" option when complate conect to Hatohol server and changed hosts information that saved in internal.
+
+ In case of using "UPDATE" option, send diffarence to use lastInfo that got using [getLastInfo](#user-content-getlastinfomethod) or saved in internal to Hatohol server.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostGroupMembership|object array|M|-|This is object array that stores host group membership information. Look at the following table for details.|
```

Comment:
> This is object array that stores host group membership information. 

"This is an object which contains host group membership information." is more natural.